src/js/disability-benefits/utils/appeal-helpers.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+export const appealStatusDescriptions = (eventType) => {
```

Comment:
Object worked—in addition to importing React, needed to change the file extension to `jsx`